keyboards/wolf/silhouette/readme.md

```diff
@@ -0,0 +1,19 @@
+# Silhouette
+
+The following is the QMK Firmware for the Silhouette PCB.
+
+The PCB features:
+* QMK & VIA compatibility
+
+---
+
+* Keyboard Maintainer: [ToastyStoemp](https://github.com/ToastyStoemp)
+
+Make example for this keyboard (after setting up your build environment):
+
+	make wolf/silhouette:default
```

Comment:
This portion should follow the [template](https://github.com/qmk/qmk_firmware/blob/master/data/templates/keyboard/readme.md?plain=1).
ie. the hardware availability and whatnot.